### ISSUE TITLE
Error invoking `rejx` on Python 3.10+

### ISSUE THREAD

**Issue by justinmayer** [MarkusSagen/rejx#2]
If I run `python -m pip install rejx` on Python 3.9, invoking `rejx` behaves as expected.

If I do the same thing on Python 3.10, 3.11, and 3.12, I get the following error. (Tap to expand traceback.)

<details>
<summary>TypeError: Name 'view' defined twice</summary>

```
$ rejx --help

╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /Users/jm/Desktop/rejxvenv310/bin/rejx:8 in <module>                                             │
│                                                                                                  │
│   5 from rejx.__main__ import app                                                                │
│   6 if __name__ == '__main__':                                                                   │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])                         │
│ ❱ 8 │   sys.exit(app())                                                                          │
│   9                                                                                              │
│                                                                                                  │
│ ╭──────────────────────────────────────── locals ─────────────────────────────────────────╮      │
│ │ app = <typer.main.Typer object at 0x1041ada20>                                          │      │
│ │  re = <module 're' from '/Users/jm/.asdf/installs/python/3.10.14/lib/python3.10/re.py'> │      │
│ │ sys = <module 'sys' (built-in)>                                                         │      │
│ ╰─────────────────────────────────────────────────────────────────────────────────────────╯      │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:328 in __call__         │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:311 in __call__         │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:350 in get_command      │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:332 in get_group        │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:483 in                  │
│ get_group_from_info                                                                              │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:577 in                  │
│ get_command_from_info                                                                            │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:553 in                  │
│ get_params_convertors_ctx_param_name_from_function                                               │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/main.py:877 in get_click_param  │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/typer/core.py:498 in __init__         │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/click/core.py:2536 in __init__        │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/click/core.py:2111 in __init__        │
│                                                                                                  │
│ /Users/jm/Desktop/rejxvenv310/lib/python3.10/site-packages/click/core.py:2653 in _parse_decls    │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: Name 'view' defined twice
```

</details>

Possibly related: https://github.com/tiangolo/typer/discussions/834

**Comment by MarkusSagen** [comment]
Solved and released new version `0.0.3`

**Comment by MarkusSagen** [comment]
Now also a `0.2.0` release
